public/stylesheets/leaguepage.css

```diff
@@ -0,0 +1,37 @@
+.container-league {
+    position: relative;
+    width: 70%;
+    margin-left: auto;
+    margin-right: auto;    
+    margin-top:5%;
+   	padding-bottom:60px;   	
```

Comment:
why are you hard coding set pixels for bottom padding. What is this padding against?